stage1_fly/aci/aci-manifest.in

```diff
@@ -32,6 +32,10 @@
         {
             "name": "coreos.com/rkt/stage1/stop",
             "value": "/stop"
+        },
+        {
+            "name": "coreos.com/rkt/stage1/interface-version",
+            "value": "4"
```

Comment:
Spoke to Iago - the only issue with bumping the version was that stage0 will no longer warn on unimplemented flags. So, I moved the warning code to stage1 and bumped the version.